src/containers/VDiskPage/VDiskPage.tsx

```diff
@@ -178,7 +178,12 @@ export function VDiskPage() {
             return (
                 <React.Fragment>
                     <div className={vDiskPageCn('storage-title')}>{vDiskPageKeyset('storage')}</div>
-                    <Storage groupId={GroupID} nodeId={nodeId} />
+                    <StorageWrapper
+                        groupId={GroupID}
+                        nodeId={nodeId}
+                        pDiskId={pDiskId ?? undefined}
+                        vDiskSlotId={vDiskSlotId ?? undefined}
+                    />
```

Comment:
If you use `PaginatedTable` on entities pages (VDisk, PDisk, StorageGroup), in most cases you know or can assume number of entities. Please pass it to `PaginatedTable` as `initialEntitiesCount`

Reply:
Came up only with the most obvious idea of predicting number of entities

https://github.com/ydb-platform/ydb-embedded-ui/pull/1397/commits/2d95afe52ad04c9667f61a69fb5398a34338fb2f#diff-523848b17ed76710cf4ba5acac5ac6d651800670770f9199339a88f94d8a4714R29


If you meant something different - give me a clue =)

Reply:
Yes, I meant something like this. For some pages we can be sure, that there would be only 1 row in table, or some value in range

NodePage  - many groups (initial count undefined), 1 node
GroupPage - 1 group, usually up to 9 nodes
PDiskPage - up to 24 groups (but usually less, 9-10 default looks fine), 1 node
VDiskPage - 1 group, 1 node

And it would be better to implement it with some helper functions:
```js
function getStorageNodesInitialEntitiesCount (viewContext){
  if(valueIsDefined(nodeId)){
    return 1
  }
  // other conditions

  return undefined
}

```